### ISSUE TITLE
In Recently Completed/Abandoned tabs, ordered by oldest first, will always bring the first PRs instead of the recent ones

### ISSUE THREAD

**Issue by jonatancompiani** [cribeiro84/azure-devops-pull-request-hub#167]
When default PR sorting is "Oldest First", the "recently completed" and "recently abandoned" tabs show always the first PRs created, instead of the recent ones.

Settings
![image](https://user-images.githubusercontent.com/15322453/107668092-09695600-6c6f-11eb-9850-4e346cc4f181.png)

Recently Completed
![image](https://user-images.githubusercontent.com/15322453/107667977-ea6ac400-6c6e-11eb-9174-096f2b04fae1.png)

**Comment by ADBjester** [comment]
Agreed.   When I visit the list of recently completed PR's, and use the "When" column to sort, I expect the sorting order (and perhaps the date itself) to be the date of COMPLETION, not the date of PR creation.   When I go here, I'm usually looking for the last PR I completed, but if that PR was an older one, it may not even be on the first page.
